integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/StreamingTableStrategy.scala

```diff
@@ -54,10 +54,10 @@ private[sql] class StreamingTableStrategy(sparkSession: SparkSession) extends Sp
           new TableIdentifier(model.tableName, model.databaseName),
           "Alter table drop column")
         Nil
-      case CarbonAlterTableDataTypeChangeCommand(model) =>
+      case CarbonAlterTableColRenameDataTypeChangeCommand(model, _) =>
         rejectIfStreamingTable(
           new TableIdentifier(model.tableName, model.databaseName),
-          "Alter table change datatype")
+          "Alter table change datatype or column rename")
```

Comment:
I think you can know whether it is rename by `CarbonAlterTableColRenameDataTypeChangeCommand(model, rename)`, so you can use different message